obytes/app-icon-badge
# Refactor Plugin + add missed typing

This PR added a simple Expo app demo to test the plugin within the repository. I also had the opportunity to test the refactoring done in #34, but it appears that it is not functioning in a real-world application. The plugin is not adding a badge to the app icon.

This is a return to the old behavior until we find the best option to fix it.

For people want to test here is how 

1. Clone the branch.  
2. Run `pnpm install`.  
3. Execute `pnpm run prebuild-demo`.  
4. Check the native folder inside the demo for icons with badges at `demo/ios/demo/Images.xcassets/AppIcon.appiconset/App-Icon-1024x1024@1x.png` and `demo/android/app/src/main/res`.  
5. If you want to try the `withDangerMod` implementation, replace the function inside `app.plugin.ts` with an async function and test again.


#31
